### ISSUE TITLE
add an example of a subject with metadata that only shows up on talk with the ! in the header on Kitteh zoo

### ISSUE THREAD

**Issue by mschwamb** [zooniverse/Panoptes-Front-End#2897]
@mrniaboc and @vrooje it would be good to add an example of the new metadata option in Kitteh Zoo so it is complete.  If you have a ! in front of the header column name, then the column shows up in Talk but no in the classification interface when you hit the metadata button. There's no example or mention of that in the Kitteh Zoo tutorial. Happy to help, but not sure how to for this.

**Comment by jelliotartz** [comment]
Closing as this has not received comments or action for over a year.

**Comment by mschwamb** [comment]
This is a documentation thing. I really think that someone should do this.

**Comment by mschwamb** [comment]
sorry reopened when hitting comment so closed again.

**Comment by vrooje** [comment]
I agree, it would be useful - could do this by just adding it to the existing subjects' metadata? That's cleaner than uploading new subjects.

I will assign myself as I agree it's important and I'm the project owner, but realistically it could be a while so if someone else wants to do it, go ahead...

**Comment by mschwamb** [comment]
I reopened the ticket so you can close when you've added that in or if someone else gets to it before you.
